docs/upgrades.md

```diff
@@ -170,7 +172,7 @@ parameter to
 
 ## FOSS to "Enterprise Edition"
 
-0. Reach out to Sematic via support@sematic.dev to obtain a license for "Sematic EE."
+0. Reach out to Sematic via `support@sematic.dev` to obtain a license for "Sematic EE".
```

Comment:
```suggestion
0. Reach out to Sematic via [support@sematic.dev](mailto:support@sematic.dev) to obtain an Enterprise License.
```

No need for the monospace font for an email.

Reply:
Done.